helpers/calendar/analytics.py

```diff
@@ -119,3 +120,32 @@ def get_meetings_duration_analytics(self, query, start_date, end_date):  # noqa:
             )
             result.append(output)
         return result
+
+    def get_booked_rooms(self, query, start_date, end_date):  # noqa: E501
+        """ Get booked room per given period of time and their percentages
+         :params
+            - query
+            - start_date, end_date(Time range)
+        """
+        start_date, end_date = CommonAnalytics.convert_dates(
+            self, start_date, end_date)
+        rooms_available = CommonAnalytics.get_calendar_id_name(
+            self, query)
+        result = []
+        bookings = 0
+        for event in rooms_available:
+            all_events_in_all_rooms = CommonAnalytics.get_all_events_in_a_room(
```

Comment:
This should be `for room in . rooms_available` like the other for loop below so it is clear and descriptive.

Reply:
@kampkelly Done